data/widgets/card/list.ui

```diff
@@ -73,6 +78,17 @@
             </child>
           </object>
         </child>
+        <child>
+          <object class="GtkImage" id="checkmark">
+            <property name="can_focus">False</property>
+            <property name="visible">False</property>
+            <property name="no_show_all">True</property>
+            <property name="resource">/com/endlessm/knowledge/data/images/checkmark.png</property>
```

Comment:
Sorry for not catching this before! The checkmark should be settable in CSS using `-gtk-icon-source`. Maybe even use `object-select-symbolic` for it in the default theme, and use this PNG in the course theme.